packages/react-table/src/components/TableComposable/examples/ComposableTableEmptyState.tsx

```diff
@@ -29,7 +29,7 @@ export const ComposableTableEmptyState: React.FunctionComponent = () => (
       <Tr>
         <Td colSpan={8}>
           <Bullseye>
-            <EmptyState variant={EmptyStateVariant.small}>
+            <EmptyState variant={EmptyStateVariant.sm}>
               <EmptyStateIcon icon={SearchIcon} />
               <Title headingLevel="h2" size="lg">
                 No results found
```

Comment:
We could replace this with EmptyStateHeader as well. Then similar to an above comment, the Button a few lines down could be wrapped in EmptyStatePrimary or Actions.